src/modules/DoNothingNode.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Demo Module. Does nothing.
+ */
+module.exports = function DoNothing(options) {
```

Comment:
Hm, was this for testing purposes, or can we add a bit more to the description to explain why it's needed, so future programmers don't get confused? Thanks!